pkgs/development/python-modules/tinyhtml5/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  webencodings,
+  pytestCheckHook,
+  flit-core,
+}:
+
+buildPythonPackage rec {
+  pname = "tinyhtml5";
+  version = "2.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-CG+ZiDPaJMMAxBTZ/oHZs2j9BMudJZagCEIcvHBfz8w=";
+  };
+
+  build-system = [ flit-core ];
+  dependencies = [ webencodings ];
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  meta = with lib; {
+    description = "A tiny HTML5 parser";
```

Comment:
```suggestion
    description = "Tiny HTML5 parser";
```

Reply:
This is a literal copy of upstreams github repo description. I prefer not to editorialize them. 

If if I was looking to nit-pick on this, maybe the description in `pyproject.toml` would be a better choice.

Reply:
That is written in the contributing guide and we had several treewide PRs already fixing this, so it is not out of my nitpicking.